doc/nrf/ug_thingy91.rst

```diff
@@ -0,0 +1,251 @@
+.. _ug_thingy91:
+
+.. _thingy91_ug_intro:
+
+Working with Thingy:91
+######################
+
+The |NCS| provides support for developing on the Thingy:91 using the following inbuilt components:
+
+* nRF9160 System in Package (SiP) using the nRF9160 DK board (PCA10090)
+* nRF52840 System on Chip (SoC) using the nRF52840 DK board (PCA10056)
```

Comment:
I'm probably misunderstanding something here, just wondering why nRF9160 DK and nRF52840 DK are mentioned?

Reply:
Modified to
 * nRF9160 System in Package (SiP)
* nRF52840 System on Chip (SoC)